Cleaner/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+
+#-----------------------------------------------------------------------------
+set(MODULE_NAME Cleaner)
+
+#-----------------------------------------------------------------------------
+
+#
+# SlicerExecutionModel
+#
+find_package(SlicerExecutionModel REQUIRED)
+include(${SlicerExecutionModel_USE_FILE})
+
+#
+# ITK
+#
+set(${PROJECT_NAME}_ITK_COMPONENTS
+  ITKIOImageBase
+  ITKSmoothing
+  )
+find_package(ITK 4.6 COMPONENTS ${${PROJECT_NAME}_ITK_COMPONENTS} REQUIRED)
```

Comment:
Specify version `5` for ITK, since this module is new .. no need to worry about supporting building against older version of ITK